tracing/src/main/java/org/axonframework/extensions/tracing/SpanUtils.java

```diff
@@ -41,24 +41,28 @@
      * <p>
```

Comment:
The usages of "type" in the javadoc feels slightly off now. This gut feel is likely why you changed the method name too. I'd change the first line to something like this:
```
Resolve the name of the given {@code message} to be used as a part of an operation name.
```